solr/jmx-tlog-solr7plus.yml

```diff
@@ -0,0 +1,180 @@
+    metricNamespace: solr
+    objectName: solr:dom1=core,dom2=${collection},dom3=${shard},dom4=${replica},category=TLOG,scope=replay,name=remaining.logs
+
+    metric:
+      - name: tlog.replay.remaining.logs
+        source: Value
+        type: gauge
+        label: tlog replay remaining logs
+        description: remaining transaction logs to replay
+        unit: logs
+
+    tag:
+      - name: solr.collection
+        value: ${collection}
+
+      - name: solr.shard
+        value: ${shard}
+
+      - name: solr.replica
+        value: ${replica}
+
+      - name: solr.core
+        value: func:ConcatUsingString(_,collection,shard,replica)
+
+  - name: tlog_replay_solr7_cloud
+    metricNamespace: solr
+    objectName: solr:dom1=core,dom2=${collection},dom3=${shard},dom4=${replica},category=TLOG,scope=replay,name=ops
+
+    metric:
+      - name: tlog.replay.ops
+        source: Count
+        type: long_gauge
+        label: replayed operations
+        description: total replayed operations
+        unit: operations
+
+    tag:
+      - name: solr.collection
+        value: ${collection}
+
+      - name: solr.shard
+        value: ${shard}
+
+      - name: solr.replica
+        value: ${replica}
+
+      - name: solr.core
+        value: func:ConcatUsingString(_,collection,shard,replica)
+
+  - name: tlog_copyOverOldUpdates_solr7_cloud
+    metricNamespace: solr
+    objectName: solr:dom1=core,dom2=${collection},dom3=${shard},dom4=${replica},category=TLOG,scope=copyOverOldUpdates,name=ops
+
+    metric:
+      - name: tlog.copyOverOldUpdates
+        source: Count
+        type: long_gauge
+        label: copy over old updates
+        description: copy over old updates requests
+        unit: requests
+
+    tag:
+      - name: solr.collection
+        value: ${collection}
+
+      - name: solr.shard
+        value: ${shard}
+
+      - name: solr.replica
+        value: ${replica}
+
+      - name: solr.core
+        value: func:ConcatUsingString(_,collection,shard,replica)
+
+  - name: tlog_buffered_solr7_cloud
+    metricNamespace: solr
+    objectName: solr:dom1=core,dom2=${collection},dom3=${shard},dom4=${replica},category=TLOG,scope=buffered,name=ops
+
+    metric:
+      - name: tlog.buffered
+        source: Value
+        type: long_gauge
+        label: buffered updates
+        description: buffered updates
+        unit: updates
+
+    tag:
+      - name: solr.collection
+        value: ${collection}
+
+      - name: solr.shard
+        value: ${shard}
+
+      - name: solr.replica
+        value: ${replica}
+
+      - name: solr.core
+        value: func:ConcatUsingString(_,collection,shard,replica)
+
+  - name: tlog_applyingBuffered_solr7_cloud
+    metricNamespace: solr
+    objectName: solr:dom1=core,dom2=${collection},dom3=${shard},dom4=${replica},category=TLOG,scope=applyingBuffered,name=ops
+
+    metric:
+      - name: tlog.applyingBuffered
+        source: Count
+        type: long_gauge
```

Comment:
Double check if this is a gauge or a counter (it might be gauge here, but better to be sure).

Reply:
it's a counter, thanks for pointing it out!